build_system/src/test.rs

```diff
@@ -908,28 +830,17 @@ fn should_remove_test(file_path: &Path) -> Result<bool, String> {
         if line.is_empty() {
             continue;
         }
-        if [
-            "//@ error-pattern:",
-            "//@ build-fail",
-            "//@ run-fail",
-            "-Cllvm-args",
-            "//~",
-            "thread",
-        ]
-        .iter()
-        .any(|check| line.contains(check))
+        if ["//@ error-pattern:", "//@ build-fail", "//@ run-fail", "-Cllvm-args", "//~", "thread"]
```

Comment:
I would prefer to keep the original formatting, so please use `#[rustfmt::skip]` or whatever.

Reply:
Sounds good to me.